test/services/postcard.spec.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+var expect  = require('chai').expect;
+var Angular = require('angular');
+var Sinon   = require('sinon');
+
+var Config  = require('../../config');
```

Comment:
nit: in addition to pallavi's comments above, this should be aligned with `=` in its own group